cmd/issue_edit.go

```diff
@@ -26,7 +26,8 @@ var issueEditCmd = &cobra.Command{
 		lab issue edit 14 -m "new title" -m "new desc"
 		lab issue edit 14 -l new_label --unlabel old_label
 		lab issue edit --milestone "NewYear"
-		lab issue edit --force-linebreak`),
+		lab issue edit --force-linebreak
+		lab issue edit 14 --delete-note <issueID>:<noteID>`),
```

Comment:
The `14` already is the `<issueID>` isn't it? Just remove it. The same is valid for the mr_edit.go message.

What about using "real" values instead of the placeholders? 
For instance: `lab issue edit --delete-note 14:736152913`

Reply:
Oh duh.  Yes.  Fixed.